### PR TITLE
add autosens prefs for ratio limits and if targets should be adjusted

### PR DESCRIPTION
**Pull request by jasoncalabrese**

### CODE REVIEW

File: lib/profile/index.js [hunk 1]
```diff
@@ -5,14 +5,17 @@ var isf = require('./isf');
 var carb_ratios = require('./carbs');
 
 function defaults ( ) {
-    var profile = {        
-          max_iob: 0 // if max_iob is not provided, never give more insulin than the pump would have
-        // , dia: pumpsettings_data.insulin_action_curve        
-        , type: "current"
```
Comment (scottleibrand) [inline lib/profile/index.js]: Not related to your PR, but does anyone know what this type: "current" does?

**Comment by scottleibrand** [comment]
I like it.  What testing have you done so far?

File: lib/profile/index.js (hunk 1 above)
Comment (jasoncalabrese) [inline lib/profile/index.js]: I was wondering what that was too

**Comment by jasoncalabrese** [comment]
I'm running it in production now, and allowing autosens to adjust targets again since it's easy to disable if it becomes a problem.  Only part I haven't seen get triggered is the logging when autosens rate gets limited.

**Comment by scottleibrand** [comment]
Cool.  Merging.

**Pull request merged**